### PR TITLE
Address sporadic failure of WebSocketIT.clientShouldReceiveDataOverLo…

### PR DESCRIPTION
**Pull request by sanjay-saxena**
…ngpollingRequest() observed only on Travis.

### CODE REVIEW

File: ws/ws/src/main/java/org/kaazing/net/impl/util/BlockingQueueImpl.java [hunk 1]
```diff
@@ -46,11 +46,15 @@ public BlockingQueueImpl() {
     }
 
     public synchronized void done() {
+        System.out.println(String.format("done(): ThreadId: %d: Connection closed; size = '%d'",
```
Comment (brennangaunce) [inline ws/ws/src/main/java/org/kaazing/net/impl/util/BlockingQueueImpl.java]: Do you mean to have println()'s in here?

File: ws/ws/src/main/java/org/kaazing/net/ws/impl/io/WsMessageReaderImpl.java [hunk 2]
```diff
@@ -101,36 +101,34 @@ public CharSequence getText() throws IOException {
...
+                System.out.println(String.format("put(): ThreadId: %d: Caught InterruptedException",
+                                                 Thread.currentThread().getId()));
+                _LOG.log(Level.INFO, ex.getMessage());
             }
 
             if (_payload == null) {
-                String s = "MessageReader has been interrupted maybe the " +
-                           "connection is closed";
+                String s = "MessageReader has been interrupted maybe the connection is closed";
                 // throw new WebSocketException(s);
```
Comment (brennangaunce) [inline ws/ws/src/main/java/org/kaazing/net/ws/impl/io/WsMessageReaderImpl.java:130]: Should remove commented code.

File: ws/ws/src/test/java/org/kaazing/gateway/client/impl/autobahn/closehandling/CloseHandlingIT.java [hunk 3]
```diff
@@ -38,6 +38,7 @@
 import org.kaazing.net.ws.WebSocketMessageReader;
 import org.kaazing.net.ws.WebSocketMessageWriter;
 
+@Ignore
```
Comment (brennangaunce) [inline ws/ws/src/test/java/org/kaazing/gateway/client/impl/autobahn/closehandling/CloseHandlingIT.java:41]: Why are we ignoring existing tests?

**Comment by pkhanal** [comment]
@brennangaunce, the changes are only for diagnostic purpose. This will eventually be removed once Sanjay figures out the issue and have the fix ready. Sanjay asked me to hold off the merging until the issue gets fixed.

**Comment by brennangaunce** [comment]
Yes - thanks - I covered with Sanjay separately.  Sounds reasonable.

**Pull request closed without merging**